src/user-guide/cheat-sheet.rst

```diff
@@ -0,0 +1,292 @@
+(good for debugging family inheritance)::
+
+   cylc config <path/id>
+
+   # view a specific task's configuration
+   cylc config <path/id> -i '[runtime][<task>]'
+
+   # view the workflow configuration with defaults applied
+   cylc config <path/id> --defaults
+
+Generate a graphical representation of the workflow's :term:`graph`
+(a useful tool for developing workflow graphs)::
+
+   cylc graph <path/id>
+
+   # render the graph between two cycle points
+   cylc graph <path/id> <cycle1> <cycle2>
+
+   # render the graph transposed (can make it easier to read)
+   cylc graph <path/id> --transpose
+
+   # group tasks by cycle point
+   cylc graph <path/id> --cycles
+
+   # collapse tasks within a family (can reduce the number of tasks displayed)
+   cylc graph <path/id> --group=<family>
+
+List all tasks and families defined in a workflow::
+
+   cylc list <path/id>
+
+
+Running Rose Stem Workflows
+---------------------------
+
+Currently, Rose stem workflows are installed using a different command to
+regular workflows::
+
+   # install a rose-stem workflow
+   rose stem
+
+   # start a rose-stem workflow
+   cylc play <id>
+
+Once a workflow is installed you can run regular Cylc commands against it, e.g
+``cylc stop``.
+
+We may be able to automatically activate ``rose stem`` functionality as part
+of ``cylc install`` in the future which would allow you to install and start
+a Rose Stem workflow with ``cylc vip``.
+
+
+Interventions
+-------------
+
+You can intervene with the running a workflow, e.g. to re-run a task.
+
+Interventions are written up in :ref:`user-guide.interventions`. Here is a
+quick summary:
+
+Run or re-run a task (:ref:`more info <interventions.re-run-a-task>`)::
+
+   cylc trigger <id>//<cycle>/<task>
+
+Mark a task as "succeeded"
+(:ref:`more info <interventions.set-task-outputs>`)::
+
+   cylc set <id>//<cycle>/<task>
+
+Kill a running job::
+
+   cylc kill <id>//<cycle>/<task>
+
+
+Running Workflows in Debug Mode
+-------------------------------
+
+When a workflow is in debug mode, more information gets written to the
+workflow's :ref:`log file <troubleshooting.log_files>`.
+Jobs also get run in Bash "xtrace" mode (``set -x``) which can help to diagnose
+the line in a task's script that caused the error.
+
+Start a workflow in debug mode::
+
+   $ cylc vip --debug <name/path>
+
+   # OR
+   $ cylc play --debug <id>
+
+Switch an already running workflow into debug mode::
+
+   cylc verbosity DEBUG <workflow-id>
+
+For more information, see :ref:`troubleshooting`.
+
+
+Managing Multiple Workflows
+---------------------------
+
+Many Cylc commands can operate over multiple workflows::
+
+   # stop all workflows
+   cylc stop '*'
+
+   # pause all workflows
+   cylc pause '*'
+
+   # re-run all failed tasks in all workflows
+   cylc trigger '*//*/*:failed'
+
+The ``*`` characters in these examples are "globs", make sure you put quotes
+around them or they won't do what you're expecting.
```

Comment:
```suggestion
The ``*`` characters in these examples are "globs", make sure you put single quotes
around them or they won't do what you're expecting (because the shell will expand them).
```

Reply:
Both single and double quotes suppress shell expansion so we should be good either way.

The choice of single vs double quotes comes down to whether the user wants to use env vars in their command.